### ISSUE TITLE
SimpleForm/Suspenders fails to install due to lack of template engine

### ISSUE THREAD

**Issue by huwd** [thoughtbot/suspenders#833]
Hey folks,

I've been running into an error, it appears to be with `simple_form`, but currently breaks the suspenders install process. Is anyone else experiencing a similar issue?

# System
```bash
~ rbenv local
2.4.0
~ suspenders -v
1.44.0
~ lsb_release -r
Release:	16.10
```

## Suspenders Command
`suspenders zi --api` initially and had experienced the failure.
Later attempted to re-run with `suspenders zi -s --api` giving the following log of bundle [here](http://pastebin.com/6dPpM7h1).

## Error
Error lists
```bash
Could not find "_form.html." in any of your source paths. Your current source paths are: 
/home/user/Documents/currentProjects/zi/lib/templates/simple_form/install
/home/user/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/simple_form-3.4.0/lib/generators/simple_form/templates
```

The simple form generator requires a [template engine](https://github.com/plataformatec/simple_form/blob/master/lib/generators/simple_form/install_generator.rb) which does not appear to being passed around here...

## Suspenders or Simple Form

Running `bundle exec rails generate simple_form:install` also fails but `rails generate simple_form:install
-e erb` succeeds!

However then trying to run `suspenders zi -s --api` doesn't seem to skip that step and so the issue repeats.

**Comment by huwd** [comment]
Thinking about this, I've overcomplicated. Why would simpleform try to install if i use --api?
It's only a front-end use right?

**Comment by mike-burns** [comment]
Yup.

How about introducing a `suspenders:api` generator that only runs the generators we'd want for an API app, and then conditionally running specific generators based on whether `--api` is passed?

**Comment by stevepolitodesign** [comment]
Closed by #1135. We now raise if new `suspenders:install:web` generator is run in the context and an API only application.
